internal/impl/laravel/5.8/handler/helper/relationship_detail.go

```diff
@@ -25,3 +25,7 @@ func NewRelationshipDetail(relationShipType int, function *core.Function, foreig
 		ReferencingModel: referencingModel,
 	}
 }
+
+func (relationshipDetail *RelationshipDetail) GetRelationshipType() int {
```

Comment:
method should be named RelationshipType

Reply:
There is a conflict with RelationshipType variable in the struct and in the class gives error
![image](https://user-images.githubusercontent.com/28625586/83019628-e47f7500-a044-11ea-8db5-d4fb2dd7dec7.png)